Modules/Loadable/Markups/SubjectHierarchyPlugins/qSlicerSubjectHierarchyMarkupsPlugin.cxx

```diff
@@ -139,6 +143,10 @@ void qSlicerSubjectHierarchyMarkupsPluginPrivate::init()
   this->ToggleSelectPointAction->setObjectName("ToggleSelectPointAction");
   QObject::connect(this->ToggleSelectPointAction, SIGNAL(triggered()), q, SLOT(toggleSelectPoint()));
 
+  this->SetTerminologyAction = new QAction("Set markup terminology", q);
```

Comment:
I think "Edit" instead of "Set" would be better (similarly to "Edit properties...") and since it opens a new window, we need to add "..." to the end. So it would become "Edit markups terminology...".

The order might be improved, too. Currently, all the "point" actions are at the top, and then come all the markup actions. It would be probably better to move this option down, just below or above "Delete markup".

Reply:
"Delete markup" is singular, "Edit markups terminology..." is plural. Which one should we change to make them consistent?

Reply:
You are right, we should use "markup" on the user interface. I just got used to using "markups" everywhere in the code.